podvm/Makefile

```diff
@@ -80,12 +87,12 @@ ifdef UEFI_FIRMWARE_LOCATION
 endif
 endif
 
-
 $(IMAGE_FILE): $(BINARIES) $(FILES) setopts
 	rm -fr output
 	rm -f cloud-init.img
 	cloud-localds cloud-init.img qcow2/userdata.cfg
 	mkdir -p toupload
+	qemu-img create -f qcow2 "se-${IMAGE_FILE}" 100G
```

Comment:
So an empty base image will be needed for SE ?
Is it possible to put it under a conditional?

Reply:
Yes this se-${IMAGE_FILE} is needed for SE. 
And as we attach this empty qcow2 `se-${IMAGE_FILE}` file as a block device to the qemu s390x instance, so we need create if before packer.

Reply:
I am trying to use a condition for SE image build.

Reply:
done